test/liquid_voting/delegations_test.exs

```diff
@@ -82,17 +84,25 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:ok, %Delegation{} = delegation} = Delegations.create_delegation(args)
     end
 
-    test "create_delegation/1 with duplicate data returns error changeset" do
-      original_delegation = insert(:delegation)
+    # FAILING!!
+    # PR #109 https://github.com/liquidvotingio/api/issues/109
+    # "Allow creation of more than one proposal-specific delegation (different proposals) with same delegator and delegate"
+    # causes this failure.
+    # Since we wish to permit the existence of both a global and a proposal-specific delegation for the same delegate/delegator
+    # pair, in certain cases (see issue # 125: https://github.com/liquidvotingio/api/issues/125, for details), this test case
+    # should actually be acceptable (and this test, therefore, should be removed/ammended appropriately)
+    #
+    # test "create_delegation/1 with duplicate data returns error changeset" do
```

Comment:
You can [`@tag :skip`](https://til.hashrocket.com/posts/rzcuw0e3ba-skip-pending-tests-in-exunit)

Reply:
> I just merged Dave's test refactor on #122, so you'll need to rebase - and solve conflicts :) basically moving tests from one file to another. Would you please solve the conflicts and ping me so I can review again?
> 
> (And a couple of other comments)

Do you mean I should pull to the branch I used for this PR?